### ISSUE TITLE
deploy breaks all supporting *.js and *.css files

### ISSUE THREAD

**Issue by karlg100** [masalinas/node-red-contrib-chartjs#15]
after deploying changes via node-red, charts break because the supporting files are no longer served via node-red.

**Comment by masalinas** [comment]
What versión are you using?

**Comment by karlg100** [comment]
node-red v0.20.5 and node-red-contrib-chartjs v0.4.1.

I've done a little debugging, and it appears that node-red deregisters the static folders when performing a Deploy if they are registered to the root URI.  I was able to fix the issue by mapping the folders to something other than /.
```
    // add static folders
    app.use('/chartjs/css', serveStatic(path.join(__dirname, "css")));
    app.use('/chartjs/js', serveStatic(path.join(__dirname, "js")));
    app.use('/chartjs/templates', serveStatic(path.join(__dirname, "templates")));
```

Of course the html templates have to be updated to map to the same and chart paths can be more than one depth.

I've only updated and fixed the line chart as that's the one I need immediately to work.  I'm planning on generating a pull request once I have time to fix the others.
